hummingbird/ml/_executor.py

```diff
@@ -67,15 +67,15 @@ def _fix_var_naming(operators, names, mod="input"):
     def forward(self, *inputs):
         with torch.no_grad():
             assert len(self._input_names) == len(inputs) or (
-                type(inputs[0]) == DataFrame
+                isinstance(inputs[0], DataFrame)
                 and DataFrame is not None
                 and not self.check_dataframe_to_array
                 and len(self._input_names) == len(inputs[0].columns)
             ), "number of inputs or number of columns in the dataframe do not match with the expected number of inputs {}".format(
                 self._input_names
             )
 
-            if type(inputs[0]) == DataFrame and DataFrame is not None:
+            if isinstance(inputs[0], DataFrame) and DataFrame is not None:
```

Comment:
Maybe `if  DataFrame is not None and isinstance(inputs[0], DataFrame)` will solve the test errors.

Reply:
Fixed.

Reply:
I still see some test error somewhere. Let me check what is going on.